src/ui/components/ToggleSwitch.vue

```diff
@@ -1,23 +1,37 @@
 <template>
-<label class="c-toggle-switch">
-    <input
-        :id="id"
-        type="checkbox"
-        :checked="checked"
-        @change="onUserSelect($event)"
+<div class="c-toggle-switch">
```

Comment:
This should likely be part of a different branch and PR? I don't think it's specific to Conditionals so if it's required to go out sooner, it could.